src/Components/CreateImageWizard/formComponents/ReviewStep.js

```diff
@@ -169,6 +173,17 @@ const ReviewStep = () => {
           <ImageDetailsList />
         </ExpandableSection>
       )}
+      {getState()?.values?.['oscap-policy'] && (
```

Comment:
Not sure if the problem is clear from the gif, it's a tad bit laggy, sorry about that.

![review](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/e710e3c7-1e05-4599-83a8-444c8d7ae5ca)

When choosing a policy and clicking through to the Review step, everything is set up nicely. But if we use the navigation on the left to go back to OpenSCAP Compliance and change a policy the navigation on the left doesn't get disabled and allows us to go directly to the Review step again.

As the FSC and Packages are populated with the customizations on their own steps, those does not get updated and stay empty.

Reply:
That is bad! Thanks for finding it!

Reply:
Can we prevent the user to click on the review step up until FSC and packages are setup ?

Reply:
Not sure. I mean the navigation gets disabled when the user changes the target, so it should be somehow doable. But as far as I can remember we don't really disable on anything else besides when the target step gets added :thinking:

Reply:
That would be the easier way I think.

Reply:
Right now It's the FSC and Packages steps that are populating themselves by querying the data from the API if a policy has been selected. Which means that if they aren't executed, we can't get customizations in.
One way to do it, would be for the OpenSCAP step to populate the form state when the user clicks.
The other way to do it is to force the user clicking through the FSC and Packages steps...

The easier is the second one, the better is the first one I guess :sweat_smile: 
I'll figure a way to do the second option after lunch.

But I would be glad to have your thoughts @regexowl and @lucasgarfield

Reply:
I would vote for forcing the user to go through the FSC and Packages again as the values in the steps change with a newly selected policy. The user should be aware of what changes happened and review them before continuing I think  :shrug:

Reply:
Ohh, btw, note that the first time you are selecting a policy you can't click on review immediately. Because everything is grayed out. But then it's only after the first time you got to the review step that you are allowed to shortcut the FSC and packages steps.

Reply:
So I guess It's not that much blocking

Reply:
I guess, the correct thing to do is: if a user goes back to the Compliance step, then we reinitialize the navigation on the left. What's your opinion about this one ?

Reply:
Yup, exactly. The steps are incrementally enabled. The problem only arises when someone wants to revisit the compliance step only and doesn't care about the customizations that are loaded.

Maybe reinitialize on change of the compliance? Could work.

Reply:
Lemme try that :)